site/guide/4-requests.md

```diff
@@ -1704,11 +1704,11 @@ Query strings are URL-encoded forms that appear in the URL of a request. Query
 parameters are declared like path parameters but otherwise handled like regular
 URL-encoded form fields. The table below summarizes the analogy:
 
-| Path Synax  | Query Syntax | Path Type Bound  | Query Type Bound |
-|-------------|--------------|------------------|------------------|
-| `<param>`   | `<param>`    | [`FromParam`]    | [`FromForm`]     |
-| `<param..>` | `<param..>`  | [`FromSegments`] | [`FromForm`]     |
-| `static`    | `static`     | N/A              | N/A              |
+| Path Syntax  | Query Syntax | Path Type Bound  | Query Type Bound |
+|--------------|--------------|------------------|------------------|
```

Comment:
Minor nitpick: we *should* be able to make this change without resizing the whole column, right?

Reply:
:man_facepalming: Yes we can. I have no idea why I thought the two trailing spaces in the column header were important, but I've pushed a commit that fixes that.